contrib/opencensus-ext-dbapi/opencensus/ext/dbapi/__init__.py

```diff
@@ -12,6 +12,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from opencensus.trace.ext.dbapi import trace
+from opencensus.ext.dbapi import trace
```

Comment:
As in https://github.com/census-instrumentation/opencensus-python/pull/529#pullrequestreview-208769605, why not move the contents of `trace.py` here?

Reply:
I was trying to avoid too many changes in this PR. Both `__init__.py` and `trace.py` were copied from the original `dbapi` folder, with the minimum change to replace namespace. We can do the clean up in a separate PR after the componentization is done.